yugabyte/run-jepsen.py

```diff
@@ -272,106 +279,99 @@ def main():
-                    num_non_zero_exit_code += 1
-
-                run_cmd(SORT_RESULTS_SH)
-
-                logging.info(
-                        "\n%s\nFinished test run #%d (%s)\n%s",
-                        "=" * 80, test_index, test_description_str, "=" * 80)
-
-                num_tests_run += 1
-                total_test_time_sec += test_elapsed_time_sec
-
-                total_elapsed_time_sec = time.time() - start_time
-                logging.info(
-                    "Finished running %d tests, "
-                    "in %d tests everything looks good, "
-                    "in %d tests something does not look good, "
-                    "%d tests timed out, "
-                    "%d tests returned zero exit code (OK), "
-                    "%d tests returned non-zero exit code (not OK -- this may includes timeouts), "
-                    "total elapsed time: %.1f sec, "
-                    "total test time: %.1f sec, "
-                    "avg test time: %.1f",
-                    num_tests_run,
-                    num_everything_looks_good,
-                    num_not_everything_looks_good,
-                    num_timed_out_tests,
-                    num_zero_exit_code,
-                    num_non_zero_exit_code,
-                    total_elapsed_time_sec,
-                    total_test_time_sec,
-                    total_test_time_sec / num_tests_run)
-                if not_good_tests:
-                    logging.info("Tests where something does not look good:\n    %s",
-                                 "\n    ".join(not_good_tests))
+                    logging.error("File %s does not exist!", jepsen_log_file)
+
+            logging.info(
+                    "Test run #%d: elapsed_time=%.1f, returncode=%d, everything_looks_good=%s",
+                    test_index, test_elapsed_time_sec, result.returncode,
+                    result.everything_looks_good)
+            if result.everything_looks_good:
+                num_everything_looks_good += 1
+            else:
+                num_not_everything_looks_good += 1
+                not_good_tests.append(test_description_str)
+            if result.returncode == 0:
+                num_zero_exit_code += 1
+            else:
+                num_non_zero_exit_code += 1
+
+            run_cmd(SORT_RESULTS_SH)
+
+            logging.info(
+                    "\n%s\nFinished test run #%d (%s)\n%s",
+                    "=" * 80, test_index, test_description_str, "=" * 80)
+
+            num_tests_run += 1
+            total_test_time_sec += test_elapsed_time_sec
+
+            total_elapsed_time_sec = time.time() - start_time
+            logging.info("Finished running %d tests.", num_tests_run)
+            logging.info("    %d OK, %d Problems (%d Timed-Out)",
+                num_everything_looks_good, num_not_everything_looks_good, num_timed_out_tests)
+            logging.info("    %d exited success, %d exited failure.", num_zero_exit_code,
```

Comment:
Nit: maybe something like `"%d test out of %d returned non-zero exit code", num_non_zero_exit_code, num_tests_run`?